Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_ctf.nut

```diff
@@ -1,5 +1,4 @@
 untyped
```

Comment:
Is `untyped` needed in this file? Which function calls here require it?

Reply:
i guess it's just because `flag.s.canTake` at this point.